pkg/controllers/clusterexternalsecret/clusterexternalsecret_controller_test.go

```diff
@@ -150,6 +150,7 @@ var _ = Describe("ClusterExternalSecret controller", func() {
 					NamespaceSelector:  metav1.LabelSelector{},
 					ExternalSecretName: ExternalSecretName,
 					ExternalSecretSpec: esv1beta1.ExternalSecretSpec{
+						RefreshInterval: &metav1.Duration{Duration: 100 * time.Millisecond},
```

Comment:
I want to shorten the refresh interval for `noMatchingNS` since the test requires a reconciliation run at least once before checkClusterExternalSecret